src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.CrossGen.targets

```diff
@@ -255,6 +255,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="_RestoreCrossgen"
           DependsOnTargets="PrepforRestoreForComposeStore;
                            _SetupStageForCrossgen;
+                           ResolveFrameworkReferences;
```

Comment:
Is it ok to add this dependency independently of the TFM the store is targeting?

Reply:
It should be, as < 3 will cause the task to execute but not output any items since there were no framework references.  Afaik, this is what we do now for any TFM since `ResolvePackageAssets` depends on this target too.